src/renderer/helpers/walletHelper.ts

```diff
@@ -44,6 +44,13 @@ export const getBnbAmountFromBalances = (balances: WalletBalances): O.Option<Ass
     O.map(({ amount }) => baseToAsset(amount))
   )
 
+export const getEthAmountFromBalances = (balances: WalletBalances): O.Option<AssetAmount> =>
+  FP.pipe(
+    balances,
+    A.findFirst(({ asset }) => isEthAsset(asset)),
+    O.map(({ amount }) => baseToAsset(amount))
+  )
+
```

Comment:
`getRuneNativeAmountFromBalances`, `getEthAmountFromBalances`, `getBnbAmountFromBalances`  are looking very similar - you might can extract common stuff out something like that:

```
export const getAssetAmountFromBalances = (balances: WalletBalances, isAsset: (asset: Asset) => boolean): O.Option<AssetAmount> =>
  FP.pipe(
    balances,
    A.findFirst(({ asset }) => isAsset(asset)),
    O.map(({ amount }) => baseToAsset(amount))
  )
  
  
 export const getEthAmountFromBalances = (balances: WalletBalances): O.Option<AssetAmount> => getAssetAmountFromBalances(balances: WalletBalances, isEthAsset)
 
 // same ^ for others

```